### ISSUE TITLE
Add Ability to Capture CPE/SWID for Product Object

### ISSUE THREAD

**Issue by ikiril01** [CybOXProject/schemas#295]
We should add the ability to specify common identifiers as part of the Product Object, namely CPEs and SWIDs.

**Comment by ikiril01** [comment]
This could be as simple as a new "Identifier" element that uses the PlatformIdentifierType from CybOX Common.

**Comment by athiasjerome** [comment]
Yes, but review current issues related to the "Platform Type", consider review of OVAL/CPE (and on top, Software as defined by Asset Identification NISTIR-7693 ai:software)
